tools/roslaunch/src/roslaunch/launch.py

```diff
@@ -540,7 +540,7 @@ def launch_node(self, node, core=False):
         except roslaunch.node_args.NodeParamsException as e:
             self.logger.error(e)
             if node.package == 'rosout' and node.type == 'rosout':
-                printerrlog("\n\n\nERROR: rosout is not built. Please run 'rosmake rosout'\n\n\n")
+                printerrlog("\n\n\nERROR: rosout is not built. Please run 'catkin_make rosout'\n\n\n")
```

Comment:
Is it a good idea to recommend calling `catkin_make` with the target `rosout`? The user could be using any other build tool (e.g. `catkin_make_isolated`, `catkin build`, etc.).